pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -820,13 +916,14 @@ func (i *OsqueryInstance) StartOsqueryClient(paths *osqueryFilePaths) (*osquery.
 
 // startOsqueryExtensionManagerServer takes a set of plugins, creates
 // an osquery.NewExtensionManagerServer for them, and then starts it.
-func (i *OsqueryInstance) StartOsqueryExtensionManagerServer(name string, socketPath string, client *osquery.ExtensionManagerClient, plugins []osquery.OsqueryPlugin) error {
+// If allowRestart is set
```

Comment:
maybe something like the errgroup will not return an exit error after successful calls to extensionManagerServer.Start(), to avoid canceling the group where restarts are expected

Reply:
Oh whoops, I don't know what happened to this comment here, thank you! Updated